com.unity.mobile.notifications/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this package will be documented in this file.
 
+## [1.0.5-preview] - 2019-03-16
```

Comment:
incorrect date it's year 2020 not 2019 :)

Reply:
Fixed at https://github.com/Unity-Technologies/com.unity.mobile.notifications/pull/15/commits/5b95b35668b7fa96204692ca2df732fc7f63cb1f